services/src/main/java/org/exoplatform/officeonline/WOPIService.java

```diff
@@ -311,7 +361,7 @@ public void start() {
     }
     discoveryPlugin.start();
 
-    LOG.debug("Office Online Editor Service started");
+    LOG.debug("WOPI Service started");
```

Comment:
Don't forget: if (LOG.isDebugEnabled....

Reply:
"WOPI Service" should reflect the class name (WOPI discovery/connector/provider will be clearer to see in the logs)